com.wps.Office.yml

```diff
@@ -20,6 +20,8 @@ finish-args:
   - --filesystem=xdg-videos
   - --filesystem=/run/media
   - --filesystem=/media
+# required for fcitx5's fcitx4frontend addon
+  - --talk-name=org.fcitx.Fcitx-99
```

Comment:
Maybe allow wildcard `org.fcitx.*` just in case?

Reply:
Done!